FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -41,6 +51,11 @@ extern NSString *const FBReferenceImageFilePathKey;
 @property (readwrite, nonatomic, assign) BOOL recordMode;
 
 /**
+ Save reference/captured/diffed images to disk after failures. Defaults to YES.
+ */
+@property (readwrite, nonatomic, assign) BOOL saveFailuresToDisk;
```

Comment:
In our case we're probably just going to use the UIImages we're sending up in the NSError, which means saving the images to disk (potentially-on device) isn't really useful work. As a hack-y example, I've kicked around the idea of wedging them into `XCActivityRecord`s so that they end up on the build machine as attachments when doing on-device testing.

I'm not married to the notion of keeping this part around, though.